pkg/executor/executor.go

```diff
@@ -22,14 +24,29 @@ func NewShellExecutor(envVars []string) Executor {
 	}
 }
 
+// Exec execute the command with the specified args
+func (e ShellExecutor) Exec(command string, args ...string) error {
+	return e.buildCmd(command, args...).Run()
+}
+
 // Exec execute the command inside the directory with the specified args
-func (e ShellExecutor) Exec(command string, dir string, args ...string) error {
+func (e ShellExecutor) ExecInDir(command string, dir string, args ...string) error {
+	cmd := e.buildCmd(command, args...)
+	if dir != "" {
+		cmd.Dir = dir
+	}
+
+	return cmd.Run()
+}
+
+func (e ShellExecutor) buildCmd(command string, args ...string) *exec.Cmd {
 	cmd := exec.Command(command, args...)
+
 	if len(e.envVars) > 0 {
 		cmd.Env = e.envVars
```

Comment:
unrelated refactoring but I feel like the `KEY=VALUE` string formatting should happen here and `envVars` should be `map[string]string`

Reply:
I have no strong feelings about this, but have done it in the last commit. LMKWYT. (stretching the propriety of acronyms there)